src/main/docs/guide/ssl.adoc

```diff
@@ -0,0 +1,33 @@
+The Cassandra DataStax Java Driver includes https://docs.datastax.com/en/developer/java-driver/latest/manual/core/ssl/#driver-configuration[support for SSL], to secure traffic between the driver and Cassandra.
+
+This an example of a configuration for Micronaut Cassandra that enables SSL connections. The relevant values are defined under the `cassandra.\*.advanced.ssl-engine-factory.*` property key and documented further in the https://docs.datastax.com/en/developer/java-driver/latest/manual/core/configuration/reference/[Datastax Configuration Reference].
+
+[configuration]
+----
+cassandra:
+  default:
+    advanced:
+      ssl-engine-factory:
+        class: DefaultSslEngineFactory
+        cipher-suites:
+          - TLS_RSA_WITH_AES_128_CBC_SHA
+          - TLS_RSA_WITH_AES_256_CBC_SHA
+        trust-store-path: <path to trust store>
+        trust-store-password: 12345
+        key-store-path: <path to key store>
+        key-store-password: 12345
```

Comment:
This is `truststore`, not `trust-store`

And `keystore`, not `key-store`

We don't need the `cypher-suites` in a minimal example either as far as I know... 🤔 

I would take the minimal example from the ssl test, ie:

```yaml
cassandra:
  default:
    advanced:
      ssl-engine-factory:
                class: DefaultSslEngineFactory
                truststore-path: <path to trust store>
                truststore-password: 12345
```

Reply:
I don't think we need any more of the config that follows